Dockerfile

```diff
@@ -36,5 +36,10 @@ RUN Rscript -e 'renv::restore()'
 ADD ./ model-res-avm/
 
 # Copy R dependencies into the app directory
-RUN mv renv model-res-avm/renv
+RUN rm -Rf model-res-avm/renv
+RUN mv renv model-res-avm/
```

Comment:
Previously this was not actually working, and was instead nesting the new `renv` directory too deep (`model-res-avm/renv/renv`); removing the old `renv` directory before moving the new one to its place resolves this issue.